README.md

```diff
@@ -350,6 +350,60 @@ module.exports = {
 
 [See resources](./resources/meilisearch-settings) for more settings examples.
 
+#### 👥 Populate entry rule
+
+Content-types in Strapi may have relationships with other content-types. To ensure that these links are fetched and added to an entry correctly from your Strapi database, the correct populate rule must be provided ([see documentation](https://docs-next.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/entity-service/populate.html#basic-populating)).
```

Comment:
Did you mean that they have relationships with each other or do they have relationships with content-type outside of Strapi? It's not really clear

Reply:
Is it better now?

Reply:
Yes !